tests/c_languages/AStyleLintBearTest.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from bears.c_languages.AStyleLintBear import AStyleLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+test_file1 = """
+int main()
+{
+    int a = 19;
+    return 0;
+}"""
```

Comment:
I see you have added a `\n` before the `"""`.  What I meant, and sorry for not being more clear, was to use
```
"""
...
}
"""
```